solutions/devsprint-vinicius-carvalho-3/DeliveryApp/Screens/Components/MenuItemCell.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  MenuItemCell.swift
+//  DeliveryApp
+//
+//  Created by Manoel Filho on 21/07/22.
+//
+
+import UIKit
+
+final class MenuItemCell: UIView {
+    
+    var menuItem: MenuItem? {
+        didSet{
+            self.nameLabel.text = menuItem?.name
+            self.priceLabel.text = .currencyValue(value: menuItem?.price ?? 0)
+            self.productImage.image = UIImage(named: "pizza")
+        }
+    }
+    
+    private lazy var wrapperItemStackView: UIStackView = {
+        let wrapperItemStackView = UIStackView()
+        wrapperItemStackView.alignment = .center
+        wrapperItemStackView.translatesAutoresizingMaskIntoConstraints = false
+        return wrapperItemStackView
+    }()
+    
+    private lazy var descriptionItemStackView: UIStackView = {
+        let descriptionItemStackView = UIStackView()
+        descriptionItemStackView.axis = .vertical
+        descriptionItemStackView.spacing = 5
+        descriptionItemStackView.translatesAutoresizingMaskIntoConstraints = false
+        return descriptionItemStackView
+    }()
+    
+    private lazy var nameLabel: UILabel = {
+        let nameLabel = UILabel()
+        nameLabel.text = "Title"
+        nameLabel.font = .systemFont(ofSize: 15, weight: .medium)
+        nameLabel.tintColor = .black
+        nameLabel.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
[Suggestion]
Not necessary in this case.

Reply:
Done. _translatesAutoresizingMaskIntoConstraints_ removed from all